airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1681,6 +1681,8 @@ def get_datasets_list(self, project_id=None):
         :param project_id: Google Cloud Project for which you
             try to get all datasets
         :type project_id: str
+        :param max_results: Max number of datasets excpected
```

Comment:
Better to use the same definition of the API.
max_results: The maximum number of results to return in a single response page.